README.md

```diff
@@ -47,6 +47,13 @@ cd hydrus
 git checkout -b develop origin/develop
 ```
 
+Create a virtual environment and activate it:
+```bash
+virtualenv venv --python=`which python3`

```

Comment:
as far as I remember, this requires `virtualenv` to be installed in the system.
In Python 3 the easiest way of create a venv is `python3 -m venv env_name`

Reply:
@Mec-iS Yes, you're right. Thanks for correcting 👍